.github/workflows/release.yaml

```diff
@@ -0,0 +1,30 @@
+name: Release Charts
+
+on:
+  push:
+    tags:
+      - chart-*
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Fetch history
+        run: git fetch --prune --unshallow
+
+      - name: Add dependency chart repos
+        run: |
+          helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.0.0-rc.2
```

Comment:
Where does this push the chart? A GitHub pages branch in the descheduler repo?

Reply:
Yes it does ([docs](https://github.com/helm/chart-releaser-action)).

Reply:
Looks good. Thanks.